lib/asciidoctor/jenkins/extensions/javadoc-link.rb

```diff
@@ -106,9 +137,9 @@
       classname = target.gsub(/#.*/, '')
       classurl = package.gsub(/\./, '/') + '/' + simpleclass + ".html"
 
-      classfrag = (target.include? "#") ? '#' + target.gsub(/.*#/, '') : ''
+      classfrag = (target.include? "#") ? '#' + target.gsub(/.*#/, '').gsub(/\(\)/, '--') : ''
```

Comment:
This accounts for inconsistent `javadoc` output between releases?

Reply:
No it just allows you to do a more natural link for simple method names,
i.e
`Beck#getAll()` rather than `Beck#getAll--`